events/index.md

```diff
@@ -1,9 +0,0 @@
----
-title: Events
-category: nav
-weight: 2
-permalink: /events/
```

Comment:
NAB: ~Do we want to have this as a redirect (even if it's not on the nav)?~ -- maybe just a `redirect_from` somewhere?
e.g. https://www.farsetlabs.org.uk/events will stop working